examples/xmllib/example_excel2xml.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+from dsp_tools import excel2xml
+
+df = pd.read_csv("example_data.csv")
+
+root = excel2xml.make_root(shortcode="0001", default_ontology="onto")
+root = excel2xml.append_permissions(root)
+
+for _, row in df.iterrows():
+    # Make Resource
+    res = excel2xml.make_resource(id=row["id"], restype=row["restype"], label=row["label"])
+
+    # Check if there are files or iiif-links
+    if not pd.isna(row["file"]):
+        bitstream = excel2xml.make_bitstream_prop(path=row["file"])
+        res.append(bitstream)
+
+    if not pd.isna(row["iiif-uri"]):
+        iiif = excel2xml.make_iiif_uri_prop(iiif_uri=row["iiif-uri"])
+        res.append(iiif)
```

Comment:
The example is already quite long. I suggest simplifying it, e.g. by removing the IIIR-URI (which is quite rare, anyways)

Reply:
I will not do that here because it would be a breaking change to the example I sent them. I consider this to be more of an experimental branch.